text/0000-security-team.md

```diff
@@ -0,0 +1,181 @@
+
+### Deadlocks
+
+* Impactful security team decisions that can't be made unanimously by the security team will be made by the core team according to their decision making process
+  * If the core team is deadlocked, the decision made will be whichever decision, if any, will provide the end user with the greatest measure of security
+
+Deadlocks should be avoided at all costs. The inability of the security team to reach a unanimous decision based on the security priorities of Grin's community and core team should be a red flag to the community and core team for the health of the team. All members should be able to clearly discuss the factual merits of the options and reach a decision that is in the clear best possible interest for the security of Grin's ecosystem and users.
+
+## Example
+
+_By following the previous example in more detail, critical differences can be further explored between the current process and the proposed security team process in the handling of a unique case._
+
+_In cases where the vulnerability is so critical that it requires a [deviation from the standard](https://github.com/mimblewimble/grin/blob/master/SECURITY.md#deviations-from-the-standard), the proposed process reduces the likelihood of the vulnerability being exploited with a focused decision making process and by restricting the details to essential persons only._
+
+### Current Process
+
+* Grin's security contacts make initial assessment of the validity of the vulnerability disclosure
+* It is determined that the disclosed vulnerability is a critical inflation bug
+* _All details of the disclosure, including the steps to carry out the exploit, are shared with the entire core team_
+* With a conversation between the entire core team, a plan of action is determined and executed, following Grin's responsible disclosure policy
+  * _This conversation may take days or weeks as there is no particular decision making process for security-related issues_
+* _Once the vulnerability is mitigated the core team may optionally share some of the details with the community_
+
+### Proposed Process
+
+* Grin's security contacts, all members of the security team, make initial assessment of the validity of the vulnerability disclosure
+* It is determined that the disclosed vulnerability is a critical inflation bug
+* _A conversation and plan of action is quickly and unanimously determined and executed on by the security team, following Grin's responsible disclosure policy_
+* _Once the vulnerability is mitigated, the security team must disclose all information that is not still security-critical with the community_
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+* The core team is already busy
```

Comment:
How is this a drawback for the formation of a security team?

Reply:
Clarified here, thanks :+1: